src/app/visualize/page.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import Visualizer from "next-route-visualizer";
```

Comment:
return just the component

should just be 2 lines, one for the import and one for the default export